templates/main.php

```diff
@@ -23,16 +23,18 @@
 $userlistParams = [];
 $allGroups= [];
 foreach($_["adminGroup"] as $group) {
-	$allGroups[] = $group['name'];
+	$allGroups[] = $group;
 }
 foreach($_["groups"] as $group) {
-	$allGroups[] = $group['name'];
+	$allGroups[] = $group;
 }
-$userlistParams['subadmingroups'] = $allGroups;
+$userlistParams['subadmingroups'] = [];
 $userlistParams['allGroups'] = json_encode($allGroups);
-$items = array_flip($userlistParams['subadmingroups']);
-unset($items['admin']);
-$userlistParams['subadmingroups'] = array_flip($items);
+for($i=0; $i < count($allGroups); $i++){
```

Comment:
Note to self: need to check where this subadmin groups array ends up being used. The code here looks like it might show a lot of subadmin groups. Maybe that is OK - it will be all the groups that some user could be made a subadmin of.